flight/Caddyfile

```diff
@@ -0,0 +1,8 @@
+:80 {
```

Comment:
I'll accept this example as soon as the example is for a secure configuration. ;)